examples/requirements_develop.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+# Installs wheels for current develop version 
+import sys, os, subprocess
```

Comment:
We need to handle if we are inside virtual environment like on gen1:
https://github.com/luxonis/depthai/blob/main/install_requirements.py#L22